packages/dev/core/src/Particles/flowMap.ts

```diff
@@ -55,9 +67,14 @@ export class FlowMap {
         const localStrength = a / 255.0;
 
         FlowVector.set(fx, fy, fz);
-        FlowVector.scaleToRef(system._tempScaledUpdateSpeed * strength * localStrength, ScaledFlowVector);
+        FlowVector.scaleToRef(strength * localStrength, ScaledFlowVector);
+
+        flowable.direction.addInPlace(ScaledFlowVector); // Update IFlowable velocity
+    }
 
-        particle.direction.addInPlace(ScaledFlowVector); // Update particle velocity
+    /** @internal */
+    public _processParticle(particle: Particle, system: ThinParticleSystem, strength = 1, matrix?: Matrix) {

```

Comment:
@deltakosh  We could remove the `system` parameter and instead pass the multiplied value of `system._tempScaledUpdateSpeed * strength` from the particle system's  `_flowUpdate.process` callback. On the other hand, having a reference to the particle system function might come in handy. Keep it or leave it out?

Reply:
good point! Let's remove it

Reply:
i will merge it when this is done